### DISCUSSION TITLE
Some issues, just wondering if anyone else having them..

### DISCUSSION THREAD

**Discussion by xXxNobuoxXx** [skyline-emu/skyline#761] [Q&A] [answered]
I have a Galaxy S10+, SD 855. I did have Mario Odyssey to boot with one of the Skyline updates I just couldn't see the screen and had audio, but now when I updated it it gets to a certain FPS and cuts off.. is that normal or do I need a other version of Skyline? Thanks for the awesome work you guys do!

**Comment by PixelyIon** [comment, marked as answer]
This is expected behavior, it's just us working on making SMO work step-by-step.

**Reply by xXxNobuoxXx** [reply]
Ok, thanks a lot.. Keep up the great work! Any Nintendo Switch game on Android is a win in my book..
